epsagon_kubernetes_subscription.sh

```diff
@@ -66,30 +67,49 @@ function send_to_epsagon {
 function apply_role {
     EPSAGON_TOKEN=$1
     KUBECTL="kubectl"
```

Comment:
Can you change here and then leave the rest of the function less cluttered? also in line 71.